tests/x11/remote_desktop/windows_network_setup.pm

```diff
@@ -32,6 +32,7 @@ sub run {
     $self->run_in_powershell(cmd => 'Get-NetIPAddress -InterfaceAlias $a.name', tags => 'win-remote-desktop');
     $self->run_in_powershell(cmd => 'New-NetIPAddress -InterfaceAlias $a.name -IPAddress 10.0.2.18 -PrefixLength 24 -DefaultGateway 10.0.2.2 -Confirm:$false', tags => 'win-remote-desktop');
     approve_network_popup;
+    wait_still_screen 3;
```

Comment:
LGTM， could you please add some comments here why you add 'sleep‘ here？

Reply:
> Got it.

Reply:
Add some comments and tags.